Library/Formula/llvm.rb

```diff
@@ -116,7 +116,7 @@ class Llvm < Formula
   option "with-lldb", "Build LLDB debugger"
   option "with-python", "Build Python bindings against Homebrew Python"
   option "with-rtti", "Build with C++ RTTI"
-  option "without-assertions", "Speeds up LLVM, but provides less debug information"
+  option "with-assertions", "Provides more debug information, but slows down LLVM"
 
   deprecated_option "rtti" => "with-rtti"
   deprecated_option "disable-assertions" => "without-assertions"
```

Comment:
I think this line is safe to remove now. It won't be doing anything at least given the option is now off-by-default.

CC @mikemcquaid